src/__tests__/Tooltip-test.js

```diff
@@ -86,4 +86,90 @@ describe('Tooltip', () => {
         expect(label.props.className)
             .toExclude('mdl-tooltip--large');
     });
+
+    it('should not render any tooltip positioning css if unspecified', () => {
+        const element = (
+            <Tooltip label="my tooltip">
+                <Icon name="add" />
+            </Tooltip>
+        );
+        const output = render(element);
+
+        const label = output.props.children[1].props.children;
+
+        expect(label.props.className)
+            .toExclude('mdl-tooltip--left')
+            .toExclude('mdl-tooltip--right')
+            .toExclude('mdl-tooltip--top')
+            .toExclude('mdl-tooltip--bottom');
+    });
+
+    it('should render tooltip left css if specified', () => {
+        const element = (
+            <Tooltip label="my tooltip" position="left">
+                <Icon name="add" />
+            </Tooltip>
+        );
+        const output = render(element);
+
+        const label = output.props.children[1].props.children;
+
+        expect(label.props.className)
+            .toInclude('mdl-tooltip--left');
+    });
+
+    it('should render tooltip right css if specified', () => {
+        const element = (
+            <Tooltip label="my tooltip" position="right">
+                <Icon name="add" />
+            </Tooltip>
+        );
+        const output = render(element);
+
+        const label = output.props.children[1].props.children;
+
+        expect(label.props.className)
+            .toInclude('mdl-tooltip--right');
+    });
+
+    it('should render tooltip top css if specified', () => {
+        const element = (
+            <Tooltip label="my tooltip" position="top">
+                <Icon name="add" />
+            </Tooltip>
+        );
+        const output = render(element);
+
+        const label = output.props.children[1].props.children;
+
+        expect(label.props.className)
+            .toInclude('mdl-tooltip--top');
+    });
+
+    it('should render tooltip bottom css if specified', () => {
+        const element = (
+            <Tooltip label="my tooltip" position="bottom">
+                <Icon name="add" />
+            </Tooltip>
+        );
+        const output = render(element);
+
+        const label = output.props.children[1].props.children;
+
+        expect(label.props.className)
+            .toInclude('mdl-tooltip--bottom');
+    });
+
+    it('should display console warning if specified position is invalid', () => {
```

Comment:
Hmm.. Yep, but this one is a little bit different, because it checks the proptype. I should add a spy on it though, and see if I can get the `new Error` value without spying the console